packages/twenty-front/src/modules/object-record/object-filter-dropdown/components/ObjectFilterSelectSubMenu.tsx

```diff
@@ -0,0 +1,110 @@
+import { StyledInput } from '@/object-record/object-filter-dropdown/components/ObjectFilterDropdownFilterSelect';
+import { useFilterDropdown } from '@/object-record/object-filter-dropdown/hooks/useFilterDropdown';
+import { FilterDefinition } from '@/object-record/object-filter-dropdown/types/FilterDefinition';
+import { FilterType } from '@/object-record/object-filter-dropdown/types/FilterType';
+import { getOperandsForFilterType } from '@/object-record/object-filter-dropdown/utils/getOperandsForFilterType';
+import { getSubMenuOptions } from '@/object-record/object-filter-dropdown/utils/getSubMenuOptions';
+import { DropdownMenuHeader } from '@/ui/layout/dropdown/components/DropdownMenuHeader';
+import { DropdownMenuItemsContainer } from '@/ui/layout/dropdown/components/DropdownMenuItemsContainer';
+import { MenuItem } from '@/ui/navigation/menu-item/components/MenuItem';
+import { useState } from 'react';
+import { IconApps, IconChevronLeft, useIcons } from 'twenty-ui';
+
+type SubMenuProps = {
+  currentSubMenu: FilterType;
+  setCurrentSubMenu: (currentSubMenu: FilterType | null) => void;
+  parent: FilterDefinition | null;
+};
+
+export const SubMenu = ({
```

Comment:
use same naming for file and exported constant for better readability
```suggestion
export const ObjectFilterSelectSubMenu = ({
```